src/entity/user/User.ts

```diff
@@ -398,6 +399,15 @@ export const validateUserUpdate = async (
     );
   }
 
+  if ('timezone' in data) {
+    const isValidTimeZone = validateValidTimeZone(data.timezone);
+    if (!isValidTimeZone) {
+      throw new ValidationError(
+        JSON.stringify({ timezone: 'invalid timezone' }),
+      );
```

Comment:
We just need to check that on FE we capture this (although very unlikely to happen with a natural flow?)

Reply:
We don't at the moment. But yeah, it wouldn't happen in a natural flow.
I was just looking at it, I can add a error toast to it in FE.

Reply:
Ah since it's already a edge-case don't bother for now I think :)